apps/ios/Shared/Views/ChatList/ChatListView.swift

```diff
@@ -106,6 +112,30 @@ struct ChatListView: View {
         }
     }
 
+    private func toggleShowUnreadButton() -> some View {
+        Button {
+            showUnread = !showUnread
+        } label: {
+            filterButtonImage(showUnread, image: "line.3.horizontal.decrease.circle", size: 22)
+        }
+    }
+
+    private func toggleShowFavouritesButton() -> some View {
+        Button {
+            showFavorites = !showFavorites
+        } label: {
+            filterButtonImage(showFavorites, image: "star", size: 24)
+        }
+    }
+
+    private func filterButtonImage(_ on: Bool, image: String, size: CGFloat) -> some View {
+        Image(systemName: image + (on ? ".fill" : ""))
```

Comment:
Better to pass as separate parameters
```suggestion
    private func filterButtonImage(_ on: Bool, image: String, imageFill: String, size: CGFloat) -> some View {
        Image(systemName: image + (on ? imageFill : image))
```